src/commands/operator/recover.ts

```diff
@@ -49,22 +69,34 @@ export default class Recover extends OperatorJobAwareCommand {
   environment!: Environment
   apiColor = color.keyword('orange')
   errorColor = color.keyword('red')
+  operatorMode: OperatorMode = OperatorMode.listen
 
   /**
    * Command Entry Point
    */
   async run(): Promise<void> {
     this.log('Loading user configurations...')
 
-    const {userWallet, configFile, supportedNetworksOptions} = await ensureConfigFileIsValid(
+    const {userWallet, configFile} = await ensureConfigFileIsValid(
       this.config.configDir,
-      undefined,
+      '748ecKCgqQkMP-KXndn9',
       true,
     )
     this.log('User configurations loaded.')
 
     const {flags} = await this.parse(Recover)
 
+    this.operatorMode =
+      OperatorMode[
+        (await checkOptionFlag(
+          Object.values(OperatorMode),
+          flags.mode,
+          'Select the mode in which to run the operator',
+        )) as keyof typeof OperatorMode
+        ]
```

Comment:
seems like some weirdness happened with formatting

Reply:
@alexanderattar yea i removed linting and pretteir for the time being. ill fix when i submit properly.